packaging/war/src/main/webapp/error.jsp

```diff
@@ -57,8 +57,8 @@ ModuleDetails shareServicesModule = moduleService.getModule("alfresco-share-serv
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
    <title>Alfresco</title>
-   <link rel="stylesheet" type="text/css" href="./css/reset.css" />
-   <link rel="stylesheet" type="text/css" href="./css/alfresco.css" />
+   <link rel="stylesheet" type="text/css" href="packaging/war/src/main/webapp/css/reset.css" />
```

Comment:
have you tested this and are you sure this is actually working?
this path doesn't mean anything for the exploded war on tomcat for example